ush/hpssarch_gen.sh

```diff
@@ -368,13 +368,16 @@ if [[ ${type} == "gdas" ]]; then
         echo "./logs/${PDY}${cyc}/gdas${fstep}.log"
       fi
     done
-    echo "./logs/${PDY}${cyc}/gdaspost*.log"
+    echo "./logs/${PDY}${cyc}/gdas*prod*.log"
+    if [[ "${WRITE_DOPOST}" == ".false." ]]; then
+       echo "./logs/${PDY}${cyc}/gdas*upp*.log"
+    fi
 
     fh=0
     while [[ ${fh} -le 9 ]]; do
       fhr=$(printf %03i "${fh}")
       echo "${COM_ATMOS_MASTER/${ROTDIR}\//}/${head}sfluxgrbf${fhr}.grib2"
-      echo "${COM_ATMOS_MASTER/${ROTDIR}\//}/${head}sfluxgrbf${fhr}.grib2.idx"
+      #echo "${COM_ATMOS_MASTER/${ROTDIR}\//}/${head}sfluxgrbf${fhr}.grib2.idx"
```

Comment:
This can be uncommented now that the indexes are being created.

Reply:
Yep, good call.  Fixed and tested on Orion.